www/css/offline.css

```diff
@@ -0,0 +1,116 @@
+body {
```

Comment:
Looks like this file is same as style.css, is there a particular reason to have this file?